guide-us-english.tex

```diff
@@ -131,7 +131,7 @@ \subsubsection{Product Owner Team}
 	\item create an overarching vision for the product \& make it visible to the organization.
 	\item build alignment with key stakeholders to secure support for backlog implementation.
 	\item generate a single, prioritized backlog, ensuring that duplication of work is avoided.
-	\item assure impediments and technical debt issues are properly prioritized in the backlog.
+	\item assure impediments get resolved by properly prioritizing them in the backlog.
```

Comment:
On the fence about this one. Impediments are more process oriented, technical debt is more product oriented. I'm all for making the Scrum definition less IT centric, but I'd rather see an IT-free alternative of "technical debt" than removing it altogether.